src/common/aliases.cpp

```diff
@@ -37,6 +37,30 @@ void ConfigParser::addAliases(cli::CLIWrapper& cli) {
     }
   });
 
+  cli.alias("int16", "true", [&](YAML::Node& config) {
+    if(mode_ == cli::mode::training) {
+      ABORT("int16 GEMM mode is not supported for training");
+    } else {
+      config["gemm-precision"] = std::string("int16");
+    }
+  });
+
+  cli.alias("int8", "true", [&](YAML::Node& config) {
+    if(mode_ == cli::mode::training) {
+      ABORT("int8 GEMM mode is not supported for training");
+    } else {
+      config["gemm-precision"] = std::string("int8");
+    }
+  });
+
+  cli.alias("int8shift", "true", [&](YAML::Node& config) {
+    if(mode_ == cli::mode::training) {
+      ABORT("int8shift GEMM mode is not supported for training");
+    } else {
+      config["gemm-precision"] = std::string("int8shift");
+    }
+  });
```

Comment:
These options are not created in the training mode, so `marian` exacutable is broken and always produces:
```
Error: Option 'int16' is not defined so alias can not be created
```
or similar. Simply do not define aliases for the trainer, or define those options in all modes.

Reply:
Sorry about that, I didn't realise exactly the config parser works.